express/parsers/apps/espresso/parser.py

```diff
@@ -329,8 +350,8 @@ def initial_structure_strings(self) -> List[str]:
                 basis = self.txt_parser.initial_basis(self._get_file_content(pw_scf_output_file))
                 lattice = self.txt_parser.initial_lattice_vectors(self._get_file_content(pw_scf_output_file))
                 structures.append(lattice_basis_to_poscar(lattice, basis))
-            except:
-                raise
+            except Exception:
+                raise # TODO: this is vacuous
```

Comment:
Let's at least log this event, pls

Reply:
<img src="https://media4.giphy.com/media/3oz8xQQP4ahKiyuxHy/giphy.gif"/>

Reply:
BTW - from https://github.com/Exabyte-io/express/pull/107/files#diff-66f652986c79dd5f940fdbc04db47630056023d036221d4dcdbf6b58840681ccL129 it seems that we've been using `print` for that purpose

Reply:
Right we should move away from `print` for many reasons as it is just not as flexible as `logging`.

Reply:
When express is invoked as part of another application (RuPy), the print statement was adding to its logs. With the change to logger, will it still be the case?

Reply:
Not likely without `logging` configured correctly. I put `print` statements back along with the logging statements for the time being but RuPy also should have proper logging configuration rather than relying on dependencies printing to `stdout`.